stdlib/websub/src/main/ballerina/src/websub/commons.bal

```diff
@@ -665,3 +721,7 @@ type WebSubError record {
     string message = "";
 };
 
+# Represents the reason string for the `websub:HubStartupError`.
+public const HUB_STARTUP_REASON = "{ballerina/websub}HubStartupError";
```

Comment:
Can we move this error definition to the error.bal? Let's maintain that file for errors?

Reply:
Yeah added this here temporarily, until the other PR was merged.